kernels/portable/cpu/util/broadcast_util.cpp

```diff
@@ -213,10 +213,20 @@ ET_NODISCARD Error get_broadcast_target_size(
     Tensor::SizesType* out_sizes,
     const size_t out_sizes_len,
     size_t* out_dim) {
-  ET_CHECK_OR_RETURN_ERROR(
-      tensors_are_broadcastable_between(a_size, b_size),
-      InvalidArgument,
-      "Two input tensors should be broadcastable.\n");
+  if (!tensors_are_broadcastable_between(a_size, b_size)) {
```

Comment:
Maybe thrown an UNLIKELY on this, but compared to the work done during broadcasting this is probably inconsequential

```suggestion
  if (ET_UNLIKELY(!tensors_are_broadcastable_between(a_size, b_size))) {
```